wallet/test/miner_tests.cpp

```diff
@@ -0,0 +1,529 @@
+// Copyright (c) 2011-2019 The Bitcoin Core developers
```

Comment:
Do we need this file if it is entirely commented out? Are we planning to enable these tests in the future?

Reply:
I have no plans for that now. But there's no harm in keeping it and looking at it later. Your choice, though.